tests/testthat/out/tibble-space.txt

```diff
@@ -0,0 +1,6 @@
+  a    
+  <chr>
+1      
```

Comment:
@hadley: I was thinking about showing a subtle `""` here for the empty string.?

Reply:
Yeah, good idea

Reply:
And maybe include `'""'` in the tests for comparison?

Reply:
Maybe the outer quotes should get `style_subtle()`?